### PR TITLE
TextField refactor

### PR DESCRIPTION
**Pull request by AlicjaSzu**
I want to merge this change because...

### Pull Request Checklist

1. [x] All visible strings are translated with proper context.
1. [x] All data-formatting is locale-aware (dates, numbers, and so on).
1. [x] The changes are tested.
1. [x] The code is documented (docstrings, project documentation).
1. [x] Changes are mentioned in the changelog.

### CODE REVIEW

File: src/@next/components/molecules/TextField/TextField.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import React from "react";
+
+import * as S from "./styles";
+import { IProps } from "./types";
+
+import { ErrorMessage, Input } from "../../atoms";
```
Comment (mateuszkula) [inline src/@next/components/molecules/TextField/TextField.tsx]: from "@components/atoms/"

File: src/@next/components/molecules/TextField/test.tsx [hunk 2]
```diff
@@ -0,0 +1,72 @@
+import { shallow } from "enzyme";
+import "jest-styled-components";
+import React from "react";
+
+import { TextField } from ".";
+import { Input } from "../../atoms";
```
Comment (mateuszkula) [inline src/@next/components/molecules/TextField/test.tsx]: "@components/atoms"

File: src/@next/components/atoms/ErrorMessage/ErrorMessage.tsx [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+import * as S from "./styles";
+import { IFormError, IProps } from "./types";
+
+export const ErrorMessage: React.FC<IProps> = ({ errors }: IProps) =>
+  !!(errors && errors.length) && (
+    <S.ErrorMessage>
+      {errors.map((error: IFormError) => error.message).join(" ")}
```
Comment (jwm0) [inline src/@next/components/atoms/ErrorMessage/ErrorMessage.tsx]: `error: IFormError` this should be auto-typed based on errors prop, no need to be explicit here

File: src/index.tsx [hunk 4]
```diff
@@ -111,13 +114,18 @@ const startApp = async () => {
...
                             apolloClient={apolloClient}
                           >
-                            <Switch>
-                              <Route
-                                path={checkoutBaseUrl}
-                                component={CheckoutApp}
-                              />
-                              <Route component={App} />
-                            </Switch>
+                            <ThemeProvider theme={defaultTheme}>
```
Comment (jwm0) [inline src/index.tsx]: ThemeProvider is already used in `src/App`

Comment (AlicjaSzu) [reply src/index.tsx]: Yes, but now styled components are not working for checkout. I will remove it, but we should think about moving Theme Provider somewhere else.

**Pull request merged**